roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -15,7 +15,7 @@ auth service required = none
 auth client required = none
 auth supported = none
 {% endif %}
-fsid = {{ fsid }}
+{% if not mon_containerized_deployment_with_kv%}fsid = {{ fsid }}{% endif %}
```

Comment:
yes something like this please. let's keep the default with the ceph-docker ceph.conf